baseplate/observers/sentry.py

```diff
@@ -67,7 +80,12 @@ def error_reporter_from_config(raw_config: config.RawConfig, module_name: str) -
                 "environment": config.Optional(config.String, default=None),
                 "include_paths": config.Optional(config.String, default=None),
                 "exclude_paths": config.Optional(config.String, default=None),
-                "ignore_exceptions": config.Optional(config.TupleOf(config.String), default=[]),
+                "ignore_exceptions": config.Optional(
+                    config.TupleOf(config.String), default=[]
+                ),  # Depricated in favor of `additional_ignore_exception
```

Comment:
```suggestion
                ),  # Deprecated in favor of additional_ignore_exception
```

If it's deprecated, should we do a [`warn_deprecated`](https://github.com/reddit/baseplate.py/blob/develop/baseplate/lib/__init__.py#L12-L20) when it's defined?